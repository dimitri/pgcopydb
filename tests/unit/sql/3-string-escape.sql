table test_str_escape;
