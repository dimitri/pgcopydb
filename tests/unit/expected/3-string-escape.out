-[ RECORD 1 ]
id | 1
f1 | aaa  +
   | aaa
-[ RECORD 2 ]
id | 2
f1 | bbb\r+
   | bbb
-[ RECORD 3 ]
id | 3
f1 | ccc

